docs/src/pages/customization/transitions/transitions.md

```diff
@@ -2,6 +2,40 @@
 
 <p class="description">The theme key enables you to customize the durations and easings of the various transitions used across Material-UI components, and offers a utility for creating custom transitions.</p>
 
+## API
+
+### `theme.transitions.create(props, options) => transition`
+
+#### Arguments
+
+1. `props` (*String* | *String[]*): Defaults to `['all']`. Provides a CSS property, or a list of CSS properties that should be transitioned.
+2.  `options` (*Object* [optional]):
+  - `options.duration` (*String* | *Number* [optional]): Defaults to `theme.transitions.duration.standard`. Provides the duration of the transition.
+  - `options.easing` (*String* [optional]): Defaults to `theme.transitions.easing.easeInOut`. Provides the easing for the transition.
+  - `options.delay` (*String* | *Number* [optional]): Defaults to `0`. Provides the delay for the transition.
+
+#### Returns
+
+`transition`: A transition CSS value, which composes all properties which should be transitioned, together with the defined duration, easing and duration.
+
+Use the <code>theme.transitions.create()</code> helper to create consistent transitions for the elements of your UI.</p>
+
+```js
+theme.transitions.create(['background-color', 'transform']);
+```
+
+{{"demo": "pages/customization/transitions/TransitionHover.js", "defaultCodeOpen": false}}
+
+### `theme.transitions.getAutoHeightDuration(height) => duration`
+
+#### Arguments
+
+1. `height` (*Number*): The height of the component.
+
+#### Returns
+
+`duration`: The calculated duration based on the height.
+
 ## Durations
 
 You can change some or all of the duration values, or provide your own (for use in the `create()` helper). This example uses the default values which are in milliseconds. You only need to provide the keys you wish to change or add a value for.
```

Comment:
```suggestion
You can change some or all of the duration values, or provide your own for use in the `create()` helper. This example uses the default values, which are in milliseconds. You only need to provide the keys you wish to change or add.
```

Reply:
Still not totally happy with that, but I think it's a little better than the previous rewording.